src/ImageProcessor/ImageProcessor.py

```diff
@@ -1,3 +1,73 @@
-from osgeo import gdal
+from osgeo import gdal, ogr
+from .Utils import get_corner_coordinates
+import os
+import subprocess
+from .ImageProcessorConfig import ImageProcessorConfig
```

Comment:
I thought we were using lower-case naming convention for module-name with absolute import paths now..? (same for `.Utils` above)